web/admin.py

```
@@ -119,25 +121,60 @@
+            {
+                "fields": (
+                    "last_email_sent_at",
+                    "last_email_event",
+                    "last_email_event_time",
+                    "email_bounce_count",
+                    "email_delivered_count",
+                    "email_open_count",
+                    "email_click_count",
+                    "email_drop_count",
+                    "email_spam_report_count",
+                )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)